### ISSUE TITLE
Inky escapes/converts single quote causing errors if used with another template language

### ISSUE THREAD

**Issue by darklow** [zurb/inky#41]
I am using Jinja2 template language for final html rendering. 
Everything works fine until i need to use strings as function arguments.
Inky escapes/converts them causing Jinja2 error:
```html
<img src="{{ user.get_image('xs') }}">
```
Result:
```html
<img src="{{ user.image_url(&apos;xs&apos;) }}">
```

Which causes:
```TemplateSyntaxError at ... ("unexpected char u'&' at 14467",)```

Any workaround or way to stop escaping single quotes?
Thank you.

**Comment by redayoub** [comment]
I'm having the same problem, how you found a solution for that ?

**Comment by darklow** [comment]
@redayoub I didn't. Still waiting for some suggestions from devs.
Right now I just added to context already final values and I am not calling anything that requires arguments.

**Comment by rafibomb** [comment]
It has to do with the Cheerio library used. @kball has a PR pending that should resolve it https://github.com/zurb/inky/pull/37

Still needs a few tweeks to get it across the line if you have suggestions.

**Comment by asmerkin** [comment]
Solution: 

Update gulpfile.babel.js

```
// Compile layouts, pages, and partials into flat HTML files
// Then parse using Inky templates
function pages() {
  return gulp.src('src/pages/**/*.html')
    .pipe(panini({
      root: 'src/pages',
      layouts: 'src/layouts',
      partials: 'src/partials',
      helpers: 'src/helpers'
    }))
    .pipe(inky({
      cheerio: {
        decodeEntities: false
      }
    }))
    .pipe(gulp.dest('dist'));
}
```

**Comment by elendirx** [comment]
+1

**Comment by darklow** [comment]
I just tested recent `inky` version and can confirm this bug was fixed (PR #57).
Now `decodeEntities` is by default false.
Thanks.
